moveit_experimental/jog_arm/src/jog_arm/joy_to_twist.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
```

Comment:
having a Python-based bridge between message formats seems like an unnecessary latency for something you want to be very responsive (teleop). is this necessary?